src/core/components/curl.jsx

```diff
@@ -20,7 +20,7 @@ export default class Curl extends React.Component {
       <div>
         <h4>Curl</h4>
         <div className="copy-paste">
-          <textarea onFocus={this.handleFocus} readOnly="true" className="curl" style={{ whiteSpace: "normal" }} value={curl}></textarea>
+          <textarea onFocus={this.handleFocus} readOnly={true} className="curl" style={{ whiteSpace: "normal" }} value={curl}></textarea>
```

Comment:
how do we feel about just sending `readOnly` without the value? It will be treated as `true` e.g

```js
<textarea onFocus={this.handleFocus} readOnly className="curl" ... />
```

Thanks for putting this PR up!

Reply:
Hmm, I never do that personally, prefer to be explicit about attributes.